is-wds-admin.php

```diff
@@ -247,11 +247,13 @@ function wds_is_admin() {
 
 /**
  * Checks if the current user is wds_admin and has special capabilities.
- * @return boolean
+ *
+ * @since  1.0.0
+ * @return boolean Is the user WDS Admin?
  */
 function is_wds_admin() {
 	return wds_is_admin()->is_wds_admin();
 }
 
 // Kick it off.
-add_action( 'admin_init', array( wds_is_admin(), 'add_cap_if_not_exists' ) );
```

Comment:
If you're logged in, but on the frontend this won't work. We want this to fire everywhere, it was before, but only when using it as an mu plugin.